creator-node/src/serviceRegistry.js

```diff
@@ -376,6 +384,94 @@ class ServiceRegistry {
     )
   }
 
+  /**
+   * Gets the regexes for routes with route params. Used for mapping paths in metrics to track route durations
+   * @param {Object} app
+   */
```

Comment:
could you put a couple examples of input-output in the jsdoc here
and a bit more detail on what this does

Reply:
bump

Reply:
moved up